staging/src/k8s.io/apiserver/pkg/server/routes/flags.go

```diff
@@ -70,15 +73,27 @@ var indexTmpl = template.Must(template.New("index").Parse(`<html>
 <br>
 flags:<br>
 <table>
-{{range .}}
-<tr>{{.Flag}}<br>
-{{end}}
-</table>
+`))
+	if err != nil {
+		return err
+	}
+
+	for _, flag := range flags {
+		_, err := fmt.Fprintf(w, `<tr>%s<br>
+`, template.HTMLEscapeString(flag.Flag))
```

Comment:
maybe switch this to use html.EscapeString, and drop the html/template import? then we can add import guards to make sure we don't pick up new use of the `html/template` package?

the only other non-test use of html/template in kubernetes code is in cmd/util/join.go, which looks like an accidental switch from text/template to html/template in https://github.com/kubernetes/kubernetes/commit/c09d875c6c4641300a64d80b97a9b983485e7d6b ... I bet that should go back to text/template (it's rendering console output for a CLI tool...)